src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClient.java

```diff
@@ -22,4 +22,11 @@
   ClientResponse runWorkflowResponse(WorkflowInputs workflowInputs);
 
   WorkflowOutputs runWorkflow(WorkflowInputs workflowInputs);
+  
+  /**
+   * Deletes the workflow with the given id.  This action is irreversible.
+   * @param id The id of the workflow to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteWorkflowResponse(String id);
```

Comment:
Further to `ClientResponse` and `DeleteHistoryResponse`, why does deleting a workflow *only* have `ClientResponse` as a return type?